src/nidigital/system_tests/test_files/pin_levels.digilevels

```diff
@@ -0,0 +1,14 @@
+<?xml version="1.0" encoding="utf-8"?>
```

Comment:
Please remove files that are not relevant to the narrowly focused PR.

Reply:
I added system tests to validate the bug fix and these files (in test_files folder) are required for the system tests. Do you want me to move the changes for system tests to a new PR?

Reply:
Sorry, no. If the changes are testing the changes in the PR then they should remain in the same PR.

Reply:
Sorry - the retrofits to the tests that are needed because of the API changes in this PR should remain in this PR.

> and these files (in test_files folder) are required for the system tests

Are these system tests new coverage? If so, I do think it's best to do that in its own PR. We should keep PRs self-contained but narrowly focused as best practice.

Reply:
Ah, makes sense. I'll move them to a separate PR.